plugins/aea-test-autonomy/aea_test_autonomy/fixture_helpers.py

```diff
@@ -274,7 +274,7 @@ def ganache_scope_class(
 @pytest.fixture(scope="class")
 def ammnet_scope_class(
     timeout: float = 2.0,
-    max_attempts: int = 20,
+    max_attempts: int = 26,
```

Comment:
Addresses #1396. 12 extra seconds should be enough for CI.

Reply:
ok so that's why. It's an odd number to pick though

Reply:
I think it's an even :sweat_smile: